app/pages/lab/subject-set.cjsx

```diff
@@ -17,9 +17,6 @@ VALID_SUBJECT_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif', '.svg']
 INVALID_FILENAME_CHARS = ['/', '\\', ':', ',']
 MAX_FILE_SIZE = 600 * 1024
 
-announceSetChange = ->
-  apiClient.type('subject_sets').emit 'add-or-remove'
```

Comment:
#announceSetChange() called [#emptySubjectQueue() in classify.cjsx](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/project/classify.cjsx#L43). However, calling #emptySubjectQueue() here does not appear to do anything, since there is no workflowID or queue variable in this scope. 

Do you know the original purpose of #announceSetChange()? I am having trouble figuring it out, so I am questioning my decision to remove the method from subject-set.csjx in this PR.